packages/hdwallet-native/src/ethereum.ts

```diff
@@ -57,7 +57,10 @@ export function MixinNativeETHWallet<TBase extends core.Constructor<NativeHDWall
 
     async ethInitializeWallet(masterKey: Isolation.Core.BIP32.Node): Promise<void> {
       const rootNode = await Isolation.Adapters.BIP32.create(masterKey);
-      const isolatedSigner = await rootNode.derivePath(ethers.utils.defaultPath);
+
+      // hard coding this path works fine. need to figure out how to
+      // re-enable supplying a path
```

Comment:
we need to derive the rootNode from a depth 5 path with the account number at index 2. this is due to upstream requirements of ethers.js.